### PR TITLE
Customize ingest

### PR DESCRIPTION
**Pull request by kimandrews**
## Description of proposed changes

Make rabies-specific modifications to the ingest directory (which originated from the pathogen-repo-guide), following these steps:
* `git subrepo pull ingest/vendored --force` so that the parent commit hash in `ingest/vendored/.gitrepo` is correct
* Add rabies-specific config parameters
* Remove files that are not currently being used for this workflow, including files related to Entrez data-fetching and Nextclade
* Add ingest automation workflow

## Checklist

- [ ] Checks pass

### CODE REVIEW

**Review by genehack — APPROVED** [review]
n.b., reviewed only the commits with new work, not the "copy X from Y" bulk ones 😁

File: .github/workflows/ingest.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+defaults:
+  run:
+    # This is the same as GitHub Action's `bash` keyword as of 20 June 2023:
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
+    #
+    # Completely spelling it out here so that GitHub can't change it out from under us
+    # and we don't have to refer to the docs to know the expected behavior.
+    shell: bash --noprofile --norc -eo pipefail {0}
+
+on:
```
Comment (joverlee521) [inline .github/workflows/ingest.yaml:12]: I want to be explicitly clear here that this workflow does not have an automated schedule. Zika's automated workflow ([ingest-to-phylogenetic](https://github.com/nextstrain/zika/blob/573ebd7d6c6f58e26b5f02f9c7ef2473623daab2/.github/workflows/ingest-to-phylogenetic.yaml#L32)) calls on the ingest workflow and phylogenetic workflow. Once the phylogenetic workflow is set up in this repo, they can be connected the same way. 

Not asking for any changes in this PR, just wanted to make sure we're all on the same page 😄

**Review by joverlee521 — APPROVED** [review]

File: .github/workflows/ingest.yaml
[the remark this answers is not in the document]
Comment (joverlee521) [reply .github/workflows/ingest.yaml:1]: Adding in https://github.com/nextstrain/infra/pull/24.

File: ingest/defaults/config.yaml [hunk 2]
```diff
@@ -66,7 +63,7 @@ curate:
   # Currently accepts any characters because we do not have a clear standard for strain names across pathogens
   strain_regex: "^.+$"
   # Back up strain name field to use if "strain" doesn"t match regex above
-  strain_backup_fields: ["accession"]
+  strain_backup_fields: []
```
Comment (j23414) [inline ingest/defaults/config.yaml:66]: [non-blocking]

I suppose there was a reason for deleting the "accession" as a backup field for strain name?

I see that the inferred command

```bash
| ./vendored/transform-strain-names   \
 --strain-regex ^.+$  \
 --backup-fields  \
 | augur curate format-dates \
 ...
 ```
 
still runs without backup fields, so I don't have a strong opinion on keeping or dropping.

**Review by j23414 — APPROVED** [review]
I'm going to assume a subsequent PR migrating to "augur curate" subcommands will fix

```
nextstrain build ingest

#> ERROR: Records do not have the same fields! Please check your input data has the same fields.
```

Code changes look good to me!

File: ingest/defaults/config.yaml (hunk 2 above)
Comment (j23414) [reply ingest/defaults/config.yaml:66]: Nevermind, I see the reason in the commit message: https://github.com/nextstrain/rabies/pull/2/commits/618269bd8ae46183d51c319060a09707d33619e1

> We plan to use accession as the strain ID field in the phylogenetic workflow, and so we will not need to replace empty strain names.

File: .github/workflows/ingest.yaml
[the remark this answers is not in the document]
Comment (joverlee521) [reply .github/workflows/ingest.yaml:1]: Merged and [deployed changes](https://github.com/nextstrain/infra/pull/24#issuecomment-2229050985), so this should be good to go.

**Comment by joverlee521** [comment]
Deleted branch and checked the box in repo to auto-delete merged branches

Screenshot 2024-07-15 at 11 34 22 AM

(We should probably add these types of configurations to nextstrain/infra)

**Pull request merged**